klippy/extras/probe.py

```diff
@@ -127,6 +127,13 @@ def _probe(self, speed):
             if "Timeout during endstop homing" in reason:
                 reason += HINT_TIMEOUT
             raise self.printer.command_error(reason)
+        # get z compensation from axis_twist_compensation
+        axis_twist_compensation = self.printer.lookup_object(
+            'axis_twist_compensation', None)
+        z_compensation = 0 if not axis_twist_compensation \
+            else axis_twist_compensation.get_z_compensation_value(pos[0])
```

Comment:
In keeping with the style of the existing probe.py code, please break this out into a regular `if` statement - for example:
```
z_comp = 0.
if axis_twist_compensation is not None:
    z_comp = axis_twist_compensation.get_z_compensation_value(pos)
```

Please check for `None` instead of using a python truth test.  Please pass the full position (not just the x coordinate) to the axis_twist_compensation module.